pyof/v0x01/controller2switch/stats_reply.py

```diff
@@ -47,17 +69,14 @@ def unpack(self, buff):
                 header.
         """
         super().unpack(buff)
+        self._unpack_body()
+
+    def _unpack_body(self):
+        """Unpack `body` replace it by the result."""
+        if self.body_type.value == 0:
```

Comment:
If we need to check for `OFPST_DESC` here, because it have a different behavior, then check for it, and not for a 'position' (`0`). If someone change the `_type` order at any time, then we would have a problem here.